driver-adapters-wasm/pg-cfpages-basic/run.sh

```diff
@@ -7,5 +7,9 @@ pnpm install
 
 pnpm prisma generate
 
-# pnpm wrangler pages deploy . --project-name pg-cfpages-basic --node-compat | tee deployment-logs.txt
-# sleep 15
\ No newline at end of file
+# First build the functions using the `--node-compat` flag into the `_worker.js` directory
+# (Note that the functions are in the non-standard `fns` directory so that they are not confused with the output `_worker.js` directory.)
+pnpm wrangler pages functions build fns --node-compat --outdir _worker.js
+# Now deploy the _worker.js alongside the index.html asset to Pages
+pnpm wrangler pages deploy . --project-name pg-cfpages-basic -- | tee deployment-logs.txt
```

Comment:
Probably clearer to build into `build/_worker.js` and then to copy `index.html` to `build/index.html` to make clear that we are "creating" a pages compatible build in `build/` and then upload that via `wrangler pages deploy build`?